.github/workflows/run_ert2_test_data_setups.yml

```diff
@@ -34,6 +34,7 @@ jobs:
     - name: Install ERT and dependencies
       run: |
         pip install .
+        pip install .[storage]
```

Comment:
Is `ert-storage` now needed to run the ert2 examples? 🤔

Reply:
it is a bit annoying, maybe we could discuss a workaround?

Reply:
Sure, I think at least that if `ert-storage` is needed to run ert2 there is no discussion anymore on whether it should be listed as one of the requirements... I guess the alternative is to go for a strategy where `ert-storage` only is imported at need / allowed to fail together with the feature flag only being accepted if `ert-storage` is available?